samples/python/mnist/graph_spliter.py

```diff
@@ -85,7 +85,7 @@ def print_type(name, type):
 print_list('backward_intializer_names_as_input', split_graphs_info.backward_intializer_names_as_input)
 print_list('intermediate_tensor_names', split_graphs_info.intermediate_tensor_names)
 print_list('user_output_grad_names', split_graphs_info.user_output_grad_names)
-print_list('backward_output_grad_names', split_graphs_info.backward_output_grad_names)
+print_list('backward_output_grad_names_map', split_graphs_info.backward_output_grad_names_map)
```

Comment:
this file is outdated and should actually be removed. 
Let's keep this file untouched, and we can removed it in another PR.

Reply:
I guess we should keep the edit for consistency.